package.json

```diff
@@ -36,23 +36,21 @@
   },
   "devDependencies": {
     "bunyan": "^1.8.10",
-    "chai": "^4.0.2",
-    "coveralls": "^3.0.0",
-    "eslint": "^4.1.1",
+    "chai": "^4.2.0",
```

Comment:
In the future I'd probably do these dev deps in a separate PR to keep the commits easier to reason about.